src/Microsoft.DotNet.ProjectModel/ProjectContext.cs

```diff
@@ -154,39 +154,6 @@ public static IEnumerable<ProjectContext> CreateContextForEachTarget(string proj
                         .BuildAllTargets();
         }
 
-
-        /// <summary>
-        /// Creates a project context based on existing context but using runtime target
-        /// </summary>
-        /// <param name="context"></param>
-        /// <param name="runtimeIdentifiers"></param>
-        /// <returns></returns>
-
-        public ProjectContext CreateRuntimeContext(IEnumerable<string> runtimeIdentifiers)
```

Comment:
Long-term, I want to restructure things so that there is a proper hierarchy: Workspace -> Project -> ProjectContext. At that point, it'll be much easier to do this. Until then, I was really concerned about not duplicating the code. It makes it harder to generate a runtime context, but even this was just a helper method around ProjectContextBuilder.

I'd rather just leave the code in one place for RC2 and then restructure things post-RC2